### PR TITLE
General liftover

### PR DESCRIPTION
**Pull request by TureBentzin**
I have removed some unused code and add the Historical Annotation.
Everything with this Annotation should not be pushed in master.

### CODE REVIEW

**Review by DSeeLP — CHANGES_REQUESTED** [review]
Use @Deprecated instead of @HistoricalCode and provide a simple description in the doc

**Comment by TureBentzin** [comment]
> Use @deprecated instead of @HistoricalCode and provide a simple description in the doc

You have marked a developer..
And yes I agree.

**Review by TureBentzin — COMMENTED** [review]
I have removed the annotation @DSeeLP

**Review by DSeeLP — CHANGES_REQUESTED** [review]
So now the only thing left is the JavaDoc which says why it is deprecated

**Comment by TureBentzin** [comment]
> So now the only thing left is the JavaDoc which says why it is deprecated

So now is the only thing left that you can add this JavaDoc...

**Comment by DSeeLP** [comment]
Just create the JavaDoc...

File: src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/FixInventoryTab.java [hunk 1]
```diff
@@ -9,14 +9,19 @@
...
 
-import java.lang.reflect.Array;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 
+/**
+ * @deprecated
+ * @Archive This code is only here for arcive purposes.
+ * @usage no usage
```
Comment (DSeeLP) [inline src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/FixInventoryTab.java]: Remove "@Archive" and "@usage"
usage is not needed
And the message of "@Archive" can be moved to "@deprecated"

File: src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 import java.util.ArrayList;
 import java.util.List;
+/**
+ * @deprecated
+ * @Archive This code is only here for arcive purposes.
```
Comment (TureBentzin) [inline src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java]: the case will be changed to lower in future

File: src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 import org.bukkit.command.TabCompleter;
-import org.bukkit.entity.Player;
 import org.bukkit.permissions.Permission;
 
 import java.util.ArrayList;
 import java.util.List;
+/**
+ * @deprecated
+ * @Archive This code is only here for arcive purposes.
+ * @usage no usage
```
Comment (DSeeLP) [inline src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java]: Remove "@Archive" and "@usage"
usage is not needed
And the message of "@Archive" can be moved to "@deprecated"

**Review by DSeeLP — CHANGES_REQUESTED** [review]

File: src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java (hunk 3 above)
Comment (TureBentzin) [reply src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java]: If I would only use only deprecated then it isn't clear that this is in the project for archive purposes.
The usage is there to tell that it doesn't work fine and could not be used.

Comment (DSeeLP) [reply src/main/java/de/tdrstudios/lobbyplugin/tabcomplete/ConfigTab.java]: When you write this it is clear that the code should not be used and doesn't work properly
```java
/**
 @deprecated This code is only here for archive purposes and should not be used.
*/
```

**Review by DSeeLP — APPROVED** [review]

**Review by lewdfoxgirl — APPROVED** [review]

**Pull request merged**